db/schema.rb

```diff
@@ -64,6 +73,13 @@
 
   add_index "oauth_applications", ["uid"], name: "index_oauth_applications_on_uid", unique: true, using: :btree
 
+  create_table "signups", force: true do |t|
```

Comment:
This is Rails modifying `db/schema.rb` running migrations, look at https://github.com/coopdevs/katuma/pull/45/files#diff-1acd2e7e27a227829d5d14a91c863bb6L19